### DISCUSSION TITLE
John Deere Cotton Picker

### DISCUSSION THREAD

**Discussion by Cody4020** [Courseplay/Courseplay_FS22#2749] [General]
I have a question about Argo Tonho CS 690 and 770, if you use the base game AI it works fine but CP will make it stop at 95% and wait to be unload. Is there something I'm doing wrong or is it just the mods itself?

**Comment by Tensuko** [comment]
That should not happen hmm.
You use the latest version?
I will have a look later on whats going on.
Those Harvester are tricky...

**Reply by Cody4020** [reply]
Yea tired 4 or 5 of version of CP and it never changed. It only does it on that mod of picker. CP is the best thing ever for FS.  I just figured it had to do with how they modded it and it just think that it is a combine.

**Reply by Tensuko** [reply]
Oh and if you try the base game cotton harvester it does work?

**Comment by Cody4020** [comment]
Yea it works just fine. I figured that out after I submitted the post.

**Reply by Tensuko** [reply]
Well then this is a mod issue.
We will not change anything there when the basegame works fine.

**Comment by Cody4020** [comment]
Okay I totally understand. Thanks you for talking to me about it.
